kotlin/android/AmpliApp/app/src/androidTest/java/com/example/ampliapp/ExampleInstrumentedTest.kt

```diff
@@ -0,0 +1,24 @@
+package com.example.ampliapp
+
+import androidx.test.platform.app.InstrumentationRegistry
+import androidx.test.ext.junit.runners.AndroidJUnit4
+
+import org.junit.Test
+import org.junit.runner.RunWith
+
+import org.junit.Assert.*
+
+/**
+ * Instrumented test, which will execute on an Android device.
+ *
+ * See [testing documentation](http://d.android.com/tools/testing).
+ */
+@RunWith(AndroidJUnit4::class)
+class ExampleInstrumentedTest {
```

Comment:
Can you please add basic sanity tests here like in Node, Browser & iOS.

Reply:
Added

Reply:
Thank you!